src/Features/LanguageServer/Protocol/Handler/OnAutoInsert/OnAutoInsertHandler.cs

```diff
@@ -145,11 +163,28 @@ internal class OnAutoInsertHandler : IRequestHandler<LSP.DocumentOnAutoInsertPar
 
             return autoInsertChange;
 
-            static async Task<TextChange> GetCollapsedChangeAsync(BraceCompletionResult result, Document oldDocument, CancellationToken cancellationToken)
+            static async Task<SourceText> GetIndentedTextAsync(Document originalDocument, SourceText textToIndent, int locationToIndent, int amountToIndent, CancellationToken cancellationToken)
+            {
+                var documentOptions = await originalDocument.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
+                var indentText = amountToIndent.CreateIndentationString(documentOptions.GetOption(FormattingOptions.UseTabs), documentOptions.GetOption(FormattingOptions.TabSize));
+                var indentedText = textToIndent.WithChanges(new TextChange(new TextSpan(locationToIndent, 0), indentText));
+                return indentedText;
```

Comment:
do you hae tests that test tabs?  we should def have those if not.

Reply:
good call, this uncovered a small bug I needed to fix :)